exercises/practice/space-age/.meta/example.f90

```diff
@@ -0,0 +1,57 @@
+
+module orbital_constants
+  implicit none
+  
+  ! Earth: orbital period 1.0 Earth years, 365.25 Earth days, or 31557600 seconds
+  double precision, parameter, private :: earth_year_seconds = 31557600.d0
+  
+  ! A planet's orbit as measured in Earth years
+  double precision, parameter, private :: mercury_earth_years = 0.2408467d0
+  double precision, parameter, private :: venus_earth_years = 0.61519726d0
+  double precision, parameter, private :: mars_earth_years = 1.8808158d0
+  double precision, parameter, private :: jupiter_earth_years = 11.862615d0
+  double precision, parameter, private :: saturn_earth_years = 29.447498d0
+  double precision, parameter, private :: uranus_earth_years = 84.016846d0
+  double precision, parameter, private :: neptune_earth_years = 164.79132d0
+
+end module orbital_constants
+
+module space_age
+  use orbital_constants
+  implicit none
```

Comment:
(Stab in the dark) Maybe the issue is that implicit rules get re-evaluated when modules are `use`d

```suggestion
  implicit none
  use orbital_constants
```